torch/onnx/symbolic_opset9.py

```diff
@@ -2240,9 +2239,46 @@ def _std(g, input, dim, unbiased, keepdim):
 # torch.std(input, dim, keepdim=False, unbiased=True)
 def std(g, input, *args):
     if len(args) == 3:
-        return _std(g, input, *args)
+        var, _ = _var(g, input, *args)
+    else:
+        var, _ = _var(g, input, None, args[0], None)
+    return g.op("Sqrt", var)
+
+
+# Since position of optional arguments can change for std, this is a hack to find if first argument
+# is 'dim' or 'unbiased'. As shown below, 'dim' argument could be listed before 'unbiased' :
+# torch.var(input, unbiased=True)
+# torch.var(input, dim, keepdim=False, unbiased=True)
+def var(g, input, *args):
```

Comment:
could this be simplifed as return var_mean(g, input, *args)[0]

Reply:
Thanks. Yeah I think this is more about style than code optimization.
Both styles have the same complexity.